Projects/Embark/Sources/MessageBubble.swift

```diff
@@ -48,9 +63,10 @@ extension MessageBubble: Viewable {
         stylingView.alpha = 0
 
         let containerView = UIStackView()
+        containerView.axis = .vertical
         containerView.isLayoutMarginsRelativeArrangement = true
         containerView.insetsLayoutMarginsFromSafeArea = false
-        containerView.layoutMargins = UIEdgeInsets(top: 5, left: 15, bottom: 3, right: 15)
+        containerView.layoutMargins = UIEdgeInsets(top: 12, left: 15, bottom: 12, right: 15)
```

Comment:
This spacing looks insane 😅 I think we need to make it smaller

Reply:
> Codewise this looks good! I noticed that the NumberActionSet component disappears off-screen before the animation is complete also.

I couldn't replicate this... hmm